advanced-usage/index.md

```diff
@@ -0,0 +1,372 @@
+- Allows loading remote versions (including from custom docker-hub repositories)
+- Allows manually uploading docker images from the surface computer
+- If an undetected failure somehow occurs in BlueOS (or if a broken version gets
+installed) it's possible to easily roll back to a working version from
+   - on the device
+   - manual upload, or
+   - downloaded from the internet
+- If necessary, the underlying service can be accessed directly at TCP port `8081`
+   - e.g. http://blueos.local:8081
+{% end %}
+
+{% pirate() %}
+### Available Services
+
+The Available Services page provides developer access to the underlying http
+server interfaces of the services upon which BlueOS is based. Each service is
+listed with 
+- the port it is served at
+- a meaningful name
+- a web-page link using the active 
+[network configuration](../getting-started/#interface-access), and where relevant
+- its API documentation (in a live-testable form)
+- the current API version
+{% end %}
+{{ easy_image(src="available-services", width=600, class="pirate") }}
+
+{% pirate() %}
+### Bridges
+
+The Bridges page allows creating high performance links between serial devices
+that are connected to the onboard computer, to a UDP port.
+{% end %}
+{{ easy_image(src="bridges", width=600, class="pirate") }}
+{% pirate() %}
+- NOTE: UDP-based systems do not guarantee packet delivery or sequential alignment
+- Bridges to external devices will generally use the localhost IP `0.0.0.0`
+- Bridges to internal programs can use the loopback IP `127.0.0.1`
+{% end %}
+{{ easy_image(src="bridges-example", width=400, class="pirate") }}
+
+{% pirate() %}
+### File Browser
+The File Browser allows viewing, editing, downloading, and uploading BlueOS files
+{% end %}
+{{ easy_image(src="file-browser", width=600, class="pirate") }}
+{% pirate() %}
+> **NOTE:** There is 
+> [a known issue](https://github.com/bluerobotics/BlueOS-docker/issues/1004)
+> where the File Browser does not load properly with certain browsers. It is
+> fixed in the 1.1.x beta releases, but in the interim for current stable
+> releases it's possible to access the underlying service at 
+> http://blueos.local:7777
+{% end %}
+
+{% pirate() %}
+### NMEA Injector
+- Conveys GPS positions (from an NMEA device) to the vehicle via MAVLink messages
+{% end %}
+{{ easy_image(src="nmea-injector", width=600, class="pirate") }}
+{% pirate() %}
+- Setup requires a socket for the NMEA device to connect to, and a MAVLink ID for
+the component that will send the location data to the vehicle
+{% end %}
+{{ easy_image(src="nmea-example", width=400, class="pirate") }}
+
+{% pirate() %}
+### Terminal
+The terminal provides
+- A tmux session
+- Direct access into the core BlueOS docker container
+- Ready access to the tmux sessions of the core services (`CTRL+b s`)
+   - Useful for seeing logs as they update live
+   - Can kill services if necessary
+- Access to the underlying device via the `red-pill` utility
```

Comment:
- To exit from the board terminal and get back to docker, you can use the `exit` command or `CTRL+d`